lib/members.ex

```diff
@@ -47,15 +46,14 @@ defmodule GlassFactoryApi.Members do
          joined_at: "2019-01-01"
       }
   """
-  @spec get_member(String.t()) :: Member.t() | nil
-  def get_member(member_id) do
-    case ApiClient.get("members/#{member_id}") do
-      {:ok, %{status_code: 200, body: body}} ->
+  @spec get_member(String.t(), map()) :: Member.t() | nil
+  def get_member(member_id, config \\ %{}) do
+    case ApiClient.get("members/#{member_id}", config) do
+      {:ok, %{status: 200, body: body}} ->
         body
-        |> Jason.decode!()
         |> Member.to_struct()
```

Comment:
Most style guides recommend avoiding pipelines for a single call:

* https://github.com/christopheradams/elixir_style_guide#avoid-single-pipelines
* https://github.com/lexmag/elixir-style-guide#needless-pipeline